pennylane/qnode.py

```diff
@@ -100,9 +100,9 @@ class QNode:
             * ``"auto"``: The QNode automatically detects the interface from the input values of
               the quantum function.
 
-        diff_method (str or .gradient_transform): The method of differentiation to use in the created QNode.
-            Can either be a :class:`~.gradient_transform`, which includes all quantum gradient
-            transforms in the :mod:`qml.gradients <.gradients>` module, or a string. The following
+        diff_method (str or .pennylane.transforms.core.TransformDispatcher): The method of differentiation to use in
+            the created QNode. Can either be a :class:`pennylane.transforms.core.TransformDispatcher`, which includes all

```

Comment:
not sure why but these links aren't rendering. not a big deal but if you can fix it, would be cool 😎